--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -3182,7 +3182,7 @@
           You'll see a prompt to try <ph name="FEATURE_NAME">$1<ex>Google Lens</ex></ph> next time you open Chromium iOS.
         </message>
         <message name="IDS_IOS_DESKTOP_PASSWORD_PROMO_REMINDER_CONFIRMATION" desc="Body text of the promo bubble that appears after uses choose to be reminded of the Password Manager on their iOS device.">
-          Next time you open Chromium on your <ph name="DEVICE_TYPE">$1<ex>iPhone</ex></ph>, you'll see a prompt to use Google Password Manager.
+          Next time you open Chromium on your <ph name="DEVICE_TYPE">$1<ex>iPhone</ex></ph>, you'll see a prompt to use Stead Vault.
         </message>
         <message name="IDS_IOS_DESKTOP_LENS_PROMO_BUBBLE_DESCRIPTION_REMINDER" desc="The description for the iOS desktop Lens promo bubble.">
           Shop, translate and identify what you see with your camera on the Chromium iOS app
--- a/chrome/app/password_manager_ui_strings.grdp
+++ b/chrome/app/password_manager_ui_strings.grdp
@@ -25,7 +25,7 @@
     Sign in automatically
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_AUTOSIGNIN_TOGGLE_DESC" desc="Text that describes the 'Sign in automatically' functionality to users.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> remembers how you signed in and automatically signs you in when possible. When off, you'll be asked for confirmation every time.
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> remembers how you signed in and automatically signs you in when possible. When off, you'll be asked for confirmation every time.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_PASSWORDS_DESCRIPTION" desc="Description for the password section entry used for managing passwords.">
     Create, save, and manage your passwords so you can easily sign in to sites and apps. <ph name="BEGIN_LINK">&lt;a target='_blank' href='$1'&gt;</ph>Learn more<ph name="END_LINK">&lt;/a&gt;</ph>
@@ -37,7 +37,7 @@
     Add shortcut
   </message>
     <message name="IDS_PASSWORD_MANAGER_UI_ADD_SHORTCUT_DESCRIPTION" desc="The description for a banner, which allows a user to add shortcut to Password Manager.">
-    To get here quicker, add a shortcut to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>
+    To get here quicker, add a shortcut to <ph name="BRAND">$1<ex>Stead Vault</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_TRUSTED_VAULT_OPT_IN_TITLE" desc="The title for the on-device encryption banner in Password Manager settings.">
     Set up on-device encryption
@@ -115,19 +115,19 @@
     Import passwords
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_DESCRIPTION_ACCOUNT_STORE_USERS" desc="The description for the password import banner in Password Manager settings.">
-    Import passwords to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>
+    Import passwords to <ph name="BRAND">$1<ex>Stead Vault</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_DESCRIPTION_SYNCING_USERS" desc="The description for the password import banner in Password Manager settings.">
-    To import passwords to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>, select a CSV file.
+    To import passwords to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>, select a CSV file.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_DESCRIPTION_SIGNEDOUT_USERS" desc="The description for the password import banner in Password Manager settings.">
-    To import passwords to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> on this device, select a CSV file.
+    To import passwords to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> on this device, select a CSV file.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_STORE_PICKER_OPTION_DEVICE" desc="An option of the store picker shown in the dialog for importing passwords to the Password Manager">
-    Save to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> on this device
+    Save to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> on this device
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_STORE_PICKER_OPTION_ACCOUNT" desc="An option of the store picker shown in the dialog for importing passwords to the Password Manager">
-    Save to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>username@gmail.com</ex></ph>
+    Save to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> for <ph name="EMAIL">$2<ex>username@gmail.com</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_STORE_PICKER_ACCESSIBLE_NAME" desc="Accessible name of destination dropdown in the passwords import dialog">
     Select where to import your passwords
@@ -160,7 +160,7 @@
     Note is more than 1000 characters
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_CONFLICT_ACCOUNT" desc="An error message for a row that wasn't imported, because the provided credential is already stored in their Google Password Manager. It is shown on the dialog for importing passwords if some passwords weren't imported.">
-    A password for this account is already saved to your <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> (<ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>)
+    A password for this account is already saved to your <ph name="BRAND">$1<ex>Stead Vault</ex></ph> (<ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>)
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_CONFLICT_DEVICE" desc="An error message for a row that wasn't imported, because the provided credential is already stored on their device. It is shown on the dialog for importing passwords if some passwords weren't imported.">
     A password for this account is already saved on this device
@@ -210,13 +210,13 @@
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_SUCCESS_SUMMARY_ACCOUNT" desc="Text on the dialog for importing passwords that shows how many passwords have been saved to Google Password Manager, after the user has chosen the file to import passwords from.">
     {NUM_PASSWORDS, plural,
-     =1 {1 password imported to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>}
-     other {{NUM_PASSWORDS} passwords imported to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>}}
+     =1 {1 password imported to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>}
+     other {{NUM_PASSWORDS} passwords imported to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>}}
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_SUCCESS_SUMMARY_DEVICE" desc="Text on the dialog for importing passwords that shows how many passwords have been stored on the user device, after the user has chosen the file to import passwords from.">
     {NUM_PASSWORDS, plural,
-     =1 {1 password imported to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> on this device}
-     other {{NUM_PASSWORDS} passwords imported to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> on this device}}
+     =1 {1 password imported to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> on this device}
+     other {{NUM_PASSWORDS} passwords imported to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> on this device}}
   </message>
    <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_CONFLICTS_TITLE" desc="The title of a dialog, which offers the ability for the user to import passwords into Password Manager. Shown when conflicts were found in the imported file and the user needs to resolve them.">
     {NUM_PASSWORDS, plural,
@@ -224,7 +224,7 @@
      other {{NUM_PASSWORDS} existing passwords found}}
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_CONFLICTS_DESCRIPTION" desc="Text shown to the user on the dialog for import passwords, in case some conflicts were found during the import process.">
-    You already have passwords for these accounts in your <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>. If you choose to import one of the passwords below, it will replace the existing one.
+    You already have passwords for these accounts in your <ph name="BRAND">$1<ex>Stead Vault</ex></ph>. If you choose to import one of the passwords below, it will replace the existing one.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_IMPORT_CANCEL" desc="A button in the dialog for importing passwords into Password Manager. By clicking the button, the import will be canceled and the dialog will be closed.">
     Cancel import
@@ -432,22 +432,22 @@
     Weak passwords are easy to guess. Make sure you're creating strong passwords.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_SIGNED_OUT" desc="Message for the case when user isn't signed in and thus Password Check can't start.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> can check your passwords when you sign in with your Google Account
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> can check your passwords when you sign in with your Google Account
   </message>
    <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_OFFLINE" desc="Message for the case when there is no internet connection.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> can’t check your passwords against data breaches. Try checking your internet connection.
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> can’t check your passwords against data breaches. Try checking your internet connection.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_COMPROMISED_SECTION" desc="Title of the compromised passwords row when an error occured.">
     Compromised passwords
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_NO_PASSWORDS" desc="Message when there are no saved passwords.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> can check your passwords when you save them
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> can check your passwords when you save them
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_QUOTA_LIMIT" desc="Message for the case when the password check failed due quota limit with a link to run the check inside a Google Account.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> can’t check your passwords against data breaches. Try again in 24 hours.
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> can’t check your passwords against data breaches. Try again in 24 hours.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_OTHER_ERROR" desc="Message for the case when the password check failed due to unknown reason.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> can’t check your passwords against data breaches. Try again later.
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> can’t check your passwords against data breaches. Try again later.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_RUNNING_LABEL" desc="Number of passwords to be checked by Password Checkup.">
     {COUNT, plural,
@@ -528,7 +528,7 @@
   <if expr="_google_chrome">
   <then>
     <message name="IDS_PASSWORD_MANAGER_UI_CHANGE_PASSWORD_MANAGER_PIN" desc="The title of a toggle which allows users to change the password manager PIN.">
-      Change Google Password Manager PIN
+      Change Stead Vault PIN
     </message>
   </then>
   <else>
@@ -541,7 +541,7 @@
     Already changed this password?
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_EDIT_DISCLAIMER_DESCRIPTION" desc="A description for the dialog which tells the user to edit password in Password Manager if it was changed already.">
-    If so, please edit your saved password in <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> so it matches your new password.
+    If so, please edit your saved password in <ph name="BRAND">$1<ex>Stead Vault</ex></ph> so it matches your new password.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_EDIT_DISCLAIMER_TITLE" desc="A title for the dialog which asks the user passwords was changed.">
     Did you already change this password on <ph name="WEBSITE">$1<ex>groupon.de</ex></ph>?
@@ -571,12 +571,12 @@
     Notes can save up to <ph name="CHARACTER_LIMIT">$1<ex>1000</ex></ph> characters.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_AUTH_TIMED_OUT" desc="Title for the dialog which shows when the password manager authentication is expired.">
-    <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> timed out
+    <ph name="BRAND">$1<ex>Stead Vault</ex></ph> timed out
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_AUTH_TIMED_OUT_DESCRIPTION" desc="Body text for the dialog which shows when the password manager authentication is expired.">
     {1, plural,
-     =1 {To keep your passwords safe, <ph name="BRAND">{0}<ex>Google Password Manager</ex></ph> locks after 1 minute of inactivity}
-     other {To keep your passwords safe, <ph name="BRAND">{0}<ex>Google Password Manager</ex></ph> locks after # minutes of inactivity}}
+     =1 {To keep your passwords safe, <ph name="BRAND">{0}<ex>Stead Vault</ex></ph> locks after 1 minute of inactivity}
+     other {To keep your passwords safe, <ph name="BRAND">{0}<ex>Stead Vault</ex></ph> locks after # minutes of inactivity}}
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_VIEW_EXISTING_PASSWORD" desc="A link text shown together with an error message when user tried editing the username to a value which is already used for the same site. On click shows existing password editor.">
     View password
@@ -615,13 +615,13 @@
     Remove <ph name="WEBSITE">$1<ex>website.com</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_EMPTY_STATE_ACCOUNT_STORE_USERS" desc="A label shown if there are no saved passwords prompting user to import them.">
-    Saved passwords will appear here. <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> Import passwords<ph name="END_LINK">&lt;/a&gt;</ph> to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>.
+    Saved passwords will appear here. <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> Import passwords<ph name="END_LINK">&lt;/a&gt;</ph> to <ph name="BRAND">$1<ex>Stead Vault</ex></ph>.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_EMPTY_STATE_SYNCING_USERS" desc="A label shown if there are no saved passwords prompting user to import them.">
-    Saved passwords will appear here. To import passwords to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>, <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> select a CSV file.<ph name="END_LINK">&lt;/a&gt;</ph>
+    Saved passwords will appear here. To import passwords to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> for <ph name="USER_EMAIL"><ex>elisa.g.becket@gmail.com</ex>$2</ph>, <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> select a CSV file.<ph name="END_LINK">&lt;/a&gt;</ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_EMPTY_STATE_SIGNEDOUT_USERS" desc="A label shown if there are no saved passwords prompting user to import them.">
-    Saved passwords will appear here. To import passwords to <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> on this device, <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> select a CSV file.<ph name="END_LINK">&lt;/a&gt;</ph>
+    Saved passwords will appear here. To import passwords to <ph name="BRAND">$1<ex>Stead Vault</ex></ph> on this device, <ph name="BEGIN_LINK">&lt;a href="#" &gt;</ph> select a CSV file.<ph name="END_LINK">&lt;/a&gt;</ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_CHECKUP_PROMO_CARD_TITLE" desc="Title for the Password Checkup promo card.">
     Password Checkup
@@ -633,7 +633,7 @@
     Check passwords
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_WEB_PROMO_CARD_TITLE" desc="Title for the Password Manager on the web promo card.">
-    Google Password Manager on the web
+    Stead Vault on the web
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_WEB_PROMO_CARD_DESCRIPTION" desc="Description for the Password Manager on the web promo card.">
     View your passwords even when you're not using Chrome or Android by signing in to <ph name="BEGIN_LINK">&lt;a target="_blank" href="$1"&gt;</ph>passwords.google.com<ph name="END_LINK">&lt;/a&gt;</ph>
@@ -642,7 +642,7 @@
     Get here quicker
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHORTCUT_PROMO_CARD_DESCRIPTION" desc="Description for the Password Manager shortcut promo card.">
-    Add a shortcut to Google Password Manager
+    Add a shortcut to Stead Vault
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_ANY_DEVICE_PROMO_CARD_TITLE" desc="Title to encorage users to use password manager on android and iOSss.">
     Use saved passwords on any device
@@ -748,10 +748,10 @@
     Passkeys access removed
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_TITLE" desc="Title of the row that lets the user delete all their password manager data.">
-    Delete all <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> data
+    Delete all <ph name="BRAND">$1<ex>Stead Vault</ex></ph> data
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_DESCRIPTION" desc="Description of the row that lets the user delete all their password manager data.">
-    Passwords, passkeys, and other data will be permanently deleted from <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>
+    Passwords, passkeys, and other data will be permanently deleted from <ph name="BRAND">$1<ex>Stead Vault</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_DELETE_ALL" desc="Text displayed on the button that lets the user delete all their password manager data.">
     Delete data
@@ -763,13 +763,13 @@
     Data deleted
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_CONFIRMATION_TITLE" desc="Title of the confirmation dialog that lets the user delete all their password manager data.">
-    You’re about to delete your <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> data
+    You’re about to delete your <ph name="BRAND">$1<ex>Stead Vault</ex></ph> data
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_CONFIRMATION_TITLE_LOCAL" desc="Title of the confirmation dialog that lets the user delete all their password manager data saved on this device.">
-    You’re about to delete your <ph name="BRAND">$1<ex>Google Password Manager</ex></ph> data saved on this device
+    You’re about to delete your <ph name="BRAND">$1<ex>Stead Vault</ex></ph> data saved on this device
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_CONFIRMATION_DESCIPTION" desc="Description of the confirmation dialog that lets the user delete all their password manager data.">
-    If you continue, your passwords, passkeys, and other data will be permanently deleted from <ph name="BRAND">$1<ex>Google Password Manager</ex></ph>. Any accounts you created for sites or apps won’t be deleted.
+    If you continue, your passwords, passkeys, and other data will be permanently deleted from <ph name="BRAND">$1<ex>Stead Vault</ex></ph>. Any accounts you created for sites or apps won’t be deleted.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_FULL_RESET_DOMAINS_DISPLAY_ONE" desc="This message is shown alongside the passwords/passkeys count displaying example domains in case when there is only one example domain.">
     For <ph name="EXAMPLE_DOMAIN_1">$1<ex>youtube.com</ex></ph>
@@ -879,10 +879,10 @@
     <if expr="_google_chrome">
     <then>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_TITLE" desc="Title for the Relaunch Chrome promo card.">
-        Google Password Manager needs access to MacOS Keychain
+        Stead Vault needs access to MacOS Keychain
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION" desc="Description for the Relaunch Chrome promo card.">
-        To use Google Password Manager with macOS Keychain, relaunch Chrome and allow Keychain access. Your tabs will reopen after relaunching.
+        To use Stead Vault with macOS Keychain, relaunch Chrome and allow Keychain access. Your tabs will reopen after relaunching.
       </message>
     </then>
     <else>
@@ -899,10 +899,10 @@
     <if expr="_google_chrome">
     <then>
      <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_TITLE_LINUX" desc="Title for the Relaunch Chrome promo card.">
-        Google Password Manager needs more access
+        Stead Vault needs more access
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION_LINUX" desc="Description for the Relaunch Chrome promo card.">
-        To use Google Password Manager with your operating system, relaunch Chrome and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+        To use Stead Vault with your operating system, relaunch Chrome and allow access to your computer's password manager. Your tabs will reopen after relaunching.
       </message>
     </then>
     <else>
@@ -1046,10 +1046,10 @@
     Your password was found in a public data breach
   </message>
   <message name ="IDS_PASSWORD_MANAGER_UI_PASSWORD_CHANGE_LEAK_DIALOG_DETAILS_WITHOUT_PRIVACY_NOTICE" desc="The first paragraph after the title. This paragraph explains what the feature offers. * “can” is important to convey that this is an optional feature. Google Password Manager has the ability to help. * “update your account” refers to the site or app the user is trying to access. For example, the user’s amazon.com password might have been found in a data breach. Google Password Manager can update the way the user gets into their amazon.com account. * “save it in [username@gmail.com]...”: This informs the user where their data gets saved. This is an identifier for their Google Account.">
-    Google Password Manager can update your account with a strong password and save it in <ph name="EMAIL">$1<ex>user@gmail.com</ex></ph>. <ph name="PASSWORD_CHANGE">$2<ex>More about this feature and relevant data sent to Google</ex></ph>
+    Stead Vault can update your account with a strong password and save it in <ph name="EMAIL">$1<ex>user@gmail.com</ex></ph>. <ph name="PASSWORD_CHANGE">$2<ex>More about this feature and relevant data sent to Google</ex></ph>
   </message>
   <message name ="IDS_PASSWORD_MANAGER_UI_PASSWORD_CHANGE_LEAK_DIALOG_DETAILS_WITH_PRIVACY_NOTICE" desc="The first paragraph after the title. This paragraph explains what the feature offers. * “can” is important to convey that this is an optional feature. Google Password Manager has the ability to help. * “update your account” refers to the site or app the user is trying to access. For example, the user’s amazon.com password might have been found in a data breach. Google Password Manager can update the way the user gets into their amazon.com account. * “saved for you in your account [username@gmail.com]...”: This informs the user where their data gets saved. This is an identifier for their Google Account.">
-    Google Password Manager can update your account with a strong password that's saved for you in your account <ph name="EMAIL">$1<ex>user@gmail.com</ex></ph>.
+    Stead Vault can update your account with a strong password that's saved for you in your account <ph name="EMAIL">$1<ex>user@gmail.com</ex></ph>.
   </message>
   <message name ="IDS_PASSWORD_MANAGER_UI_PASSWORD_CHANGE_LEAK_DIALOG_LINK_WITHOUT_PRIVACY_NOTICE" desc="Link to a help-center article that explains the password change feature in more details.">
     More about this feature and relevant data sent to Google
@@ -1078,12 +1078,12 @@
   <if expr="_google_chrome">
   <then>
     <message name="IDS_SETTINGS_PASSWORD_CHANGE_SUBLABEL_WITH_LEARN_MORE" desc="Sublabel for password change entry in settings. The description of the feature beneath the 'Automated password change' feature title. This and the bullets beneath this paragraph represent the full description of the feature. * 'public' does two things: 1) it provides distance between 'Chrome' and 'data breach', because we don’t want users to associate the breach with Chrome (Chrome didn’t have a data leak, 2) and related, it might reassure the user to know that Chrome is using public information to establish these lists of compromised passwords. * 'Google Password Manager can offer…': The user has to agree to have this feature update their password for them each time. So Google Password Manager is ‘offering’ to do something for the user.">
-      If Chrome finds one of your passwords in a public data breach, Google Password Manager can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
+      If Chrome finds one of your passwords in a public data breach, Stead Vault can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
     </message>
   </then>
   <else>
     <message name="IDS_SETTINGS_PASSWORD_CHANGE_SUBLABEL_WITH_LEARN_MORE" desc="Sublabel for password change entry in settings. The description of the feature beneath the 'Automated password change' feature title. This and the bullets beneath this paragraph represent the full description of the feature. * 'public' does two things: 1) it provides distance between 'Chrome' and 'data breach', because we don’t want users to associate the breach with Chrome (Chrome didn’t have a data leak, 2) and related, it might reassure the user to know that Chrome is using public information to establish these lists of compromised passwords. * 'Google Password Manager can offer…': The user has to agree to have this feature update their password for them each time. So Google Password Manager is ‘offering’ to do something for the user.">
-      If Chromium finds one of your passwords in a public data breach, Google Password Manager can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
+      If Chromium finds one of your passwords in a public data breach, Stead Vault can offer to change your password for you if you're signed in. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>More about automated password change<ph name="END_LINK">&lt;/a&gt;</ph>
     </message>
   </else>
   </if>
@@ -1100,7 +1100,7 @@
     Recovery password — temporarily saved for up to 3 months
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_BACKUP_PASSWORD_SETTINGS_DESCRIPTION_DETAILS" desc="Text shown in the note section of the password details card in the password manager setting page with the specfic details of when the change happened.">
-    Your original password for this site was found in a public data breach. On <ph name="DATE">$1<ex>Mar 17</ex></ph> you asked <ph name="BRAND">$2<ex>Google Password Manager</ex></ph> to automatically change your password for you.
+    Your original password for this site was found in a public data breach. On <ph name="DATE">$1<ex>Mar 17</ex></ph> you asked <ph name="BRAND">$2<ex>Stead Vault</ex></ph> to automatically change your password for you.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_OTP_DIALOG_TITLE" desc="The title of a dialog box. Automated password change acts on the user’s behalf, working with the site to automatically change a password. But sometimes there are manual steps, like a Captcha or a one-time password, that the automated system can’t handle. This message explains that.">
     The site needs a one-time password
