--- a/chrome/app/chromium_strings.grd
+++ b/chrome/app/chromium_strings.grd
@@ -657,13 +657,13 @@
         Chromium
       </message>
       <message name="IDS_PASSWORD_BUBBLES_PASSWORD_MANAGER_LINK_TEXT_SAVING_ON_DEVICE" desc="The text of the Google Password Manager links displayed on Password Manager bubbles footer when the credentials are stored only locally.">
-        Password Manager
+        Stead Vault
       </message>
       <message name="IDS_PASSWORD_MANAGER_SAVE_PASSWORD_SIGNED_OUT_MESSAGE_DESCRIPTION" desc="The description of the Password Manager's save password message when the user is signed out.">
-        To Password Manager on this device
+        To Stead Vault on this device
       </message>
       <message name="IDS_PASSWORD_MANAGER_UPDATE_PASSWORD_SIGNED_OUT_MESSAGE_DESCRIPTION" desc="The description of the Password Manager's update password message when the user is signed out.">
-        In Password Manager on this device
+        In Stead Vault on this device
       </message>
       <if expr="is_macosx">
       <message name="IDS_PASSWORDS_AUTHENTICATION_PROMPT_PREFIX" desc="Text shown in the beginning of the OS-level user authentication prompt before performing a seneitive operation such as show passwords in plain text.">
@@ -684,7 +684,7 @@
           Chromium wants to export your passwords. Type your Windows password to allow this.
         </message>
         <message name="IDS_PASSWORDS_PAGE_DELETE_ALL_DATA_AUTHENTICATION_PROMPT" desc="Text for the dialog box that prompts the user for their OS account password before deleting all data from Google Password Manager.">
-          Chromium wants to delete your Password Manager data. Type your Windows password to allow this.
+          Stead wants to delete your Stead Vault data. Type your Windows password to allow this.
         </message>
         <message name="IDS_PASSWORDS_PAGE_IMPORT_AUTHENTICATION_PROMPT" desc="Text for the dialog box that prompts the user for their OS account password before overriding passwords during Import.">
           Chromium is trying to replace existing passwords. Type your Windows password to allow this.
@@ -733,10 +733,10 @@
       </if>
       <if expr="not is_android">
         <message name="IDS_PASSWORD_MANAGER_IMPORT_DIALOG_TITLE" desc="The title of the file selection dialog for importing passwords">
-          Import Passwords To Chromium
+          Import Passwords to Stead Vault
         </message>
         <message name="IDS_PASSWORD_MANAGER_EXPORT_DIALOG_TITLE" desc="The title of the file selection dialog for exporting passwords">
-          Export Passwords From Chromium
+          Export Passwords from Stead Vault
         </message>
       </if>
       <!-- Options Dialog -->
@@ -1122,7 +1122,7 @@
       </if>
 
       <message name="IDS_VIEW_PASSWORDS" desc="Item in the app menu which shows Password Manager page">
-        P&amp;assword Manager
+        Stead &amp;Vault
       </message>
 
       <!-- Chrome sign-in page -->
@@ -1592,13 +1592,13 @@
           Click the Chromium menu
         </message>
         <message name="IDS_TUTORIAL_PASSWORD_MANAGER_CLICK_PASSWORD_MANAGER" desc="The description of the 'click password manager' step in password manager tutorial">
-          Click “Password Manager”
+          Click “Stead Vault”
         </message>
         <message name="IDS_TUTORIAL_PASSWORD_MANAGER_SUCCESS_BODY" desc="The description of the body text in the 'success' step in password manager tutorial">
-          Use your shortcut to quickly get to the Password Manager. You can move your shortcut to your computer's home screen or app launcher.
+          Use your shortcut to quickly get to Stead Vault. You can move your shortcut to your computer's home screen or app launcher.
         </message>
         <message name="IDS_PASSWORD_MANAGER_IPH_CREATE_SHORTCUT_BODY" desc="Body of the in Product Help shown after successfully logging into a website telling user about how to create a shortcut">
-          Add a shortcut to Password Manager
+          Add a shortcut to Stead Vault
         </message>
       </if>
 
--- a/chrome/app/generated_resources.grd
+++ b/chrome/app/generated_resources.grd
@@ -7059,13 +7059,13 @@
         Passwords are saved in your Google Account so you can use them on any device
       </message>
       <message name="IDS_PASSWORD_BUBBLES_PASSWORD_MANAGER_LINK_TEXT_SYNCED_TO_ACCOUNT" desc="The text of the Google Password Manager links displayed on Password Manager bubbles footer when the credentials are synced to the Google account.">
-        Google Password Manager
+        Stead Vault
       </message>
       <message name="IDS_PASSWORD_BUBBLES_FOOTER_SYNCED_TO_ACCOUNT" desc="The footer text of the bubbles that offers users to save, update or manage stored passwords in Chrome and synced to their Google account.">
         You can use saved passwords on any device. They're saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>user@gmail.com</ex></ph>.
       </message>
       <message name="IDS_PASSWORD_BUBBLES_FOOTER_SAVING_ON_DEVICE" desc="The footer text of the bubbles that offers users to save, update or manage stored passwords only locally in Chrome.">
-        Passwords are saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google Password Manager</ex></ph> on this device.
+        Passwords are saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Stead Vault</ex></ph> on this device.
       </message>
       <message name="IDS_PASSWORD_GENERATION_PROMPT_GOOGLE_PASSWORD_MANAGER" desc="Autofill dropdown text describing password generation.">
           You won't need to remember this password. It will be saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Google Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>user@gmail.com</ex></ph>.
--- a/chrome/app/password_manager_ui_strings.grdp
+++ b/chrome/app/password_manager_ui_strings.grdp
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <grit-part>
   <message name="IDS_PASSWORD_MANAGER_UI_TITLE" desc="Title of the Password Manager page">
-    Password Manager
+    Stead Vault
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_DESCRIPTION" desc="A description of the application.">
     Keep track of your passwords and manage them in one place.
@@ -49,7 +49,7 @@
     On-device encryption
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_TRUSTED_VAULT_OPTED_IN_DESCRIPTION" desc="Sub-label for the on-device encryption banner in settings, when the user is already opted in.">
-    Your passwords are encrypted on your device before they‘re saved to Google Password Manager
+    Your passwords are encrypted on your device before they’re saved to Stead Vault
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHARE" desc="A button on the password card that triggers sharing flow.">
     Share
@@ -76,13 +76,13 @@
     For now, you can only share passwords with family members. <ph name="BEGIN_LINK">&lt;a href="$1" target="_blank"&gt;</ph>Invite family members<ph name="END_LINK">&lt;/a&gt;</ph> to join your group, and get more from your products and subscriptions across Google.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHARE_PASSWORD_FAMILY_PICKER_DESCRIPTION" desc="The descrtiption of the dialog that is shown to users when they started the password sharing flow and now they need to select the recipients.">
-    When you share a copy of your <ph name="BEGIN_BOLD_USERNAME">&lt;b&gt;</ph>username<ph name="END_BOLD_USERNAME">&lt;/b&gt;</ph> and <ph name="BEGIN_BOLD_PASSWORD">&lt;b&gt;</ph>password<ph name="END_BOLD_PASSWORD">&lt;/b&gt;</ph>, your family member can fill them using Google Password Manager
+    When you share a copy of your <ph name="BEGIN_BOLD_USERNAME">&lt;b&gt;</ph>username<ph name="END_BOLD_USERNAME">&lt;/b&gt;</ph> and <ph name="BEGIN_BOLD_PASSWORD">&lt;b&gt;</ph>password<ph name="END_BOLD_PASSWORD">&lt;/b&gt;</ph>, your family member can fill them using Stead Vault
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHARE_PASSWORD_CONFIRMATION_DESCRIPTION_SINGLE" desc="The description of the dialog that is shown to the user when they've just shared a password with a single family members.">
-    <ph name="RECIPIENT_NAME">&lt;b&gt;$1&lt;/b&gt;<ex>John Doe</ex></ph> can now use your username and password when they use Google Password Manager. Tell them to go to <ph name="WEBSITE">&lt;b&gt;$2&lt;/b&gt;<ex>website.com</ex></ph> to sign in.
+    <ph name="RECIPIENT_NAME">&lt;b&gt;$1&lt;/b&gt;<ex>John Doe</ex></ph> can now use your username and password when they use Stead Vault. Tell them to go to <ph name="WEBSITE">&lt;b&gt;$2&lt;/b&gt;<ex>website.com</ex></ph> to sign in.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHARE_PASSWORD_CONFIRMATION_DESCRIPTION_MULTIPLE" desc="The description of the dialog that is shown to the user when they've just shared a password with multiple family members.">
-    Your family members can now use your username and password when they use Google Password Manager. Tell them to go to <ph name="WEBSITE">&lt;b&gt;$1&lt;/b&gt;<ex>website.com</ex></ph> to sign in.
+    Your family members can now use your username and password when they use Stead Vault. Tell them to go to <ph name="WEBSITE">&lt;b&gt;$1&lt;/b&gt;<ex>website.com</ex></ph> to sign in.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_SHARE_PASSWORD_CONFIRMATION_FOOTER_WEBSITE" desc="The information about how to stop sharing the password in the footer of confirmation dialog which is shown to the user when they've just shared a password.">
     To stop others from using your password, change it on <ph name="WEBSITE">$1<ex>&lt;a href="https://website.com/" target="_blank"&gt;website.com&lt;/a&gt;</ex></ph>
--- a/components/password_manager_strings.grdp
+++ b/components/password_manager_strings.grdp
@@ -17,22 +17,22 @@
     Check your saved passwords
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHECK_PASSWORDS_MESSAGE_GPM_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials are saved and used on multiple sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, branded.">
-    The password you just used was found in a data breach. To secure your accounts, Google Password Manager recommends checking your saved passwords.
+    The password you just used was found in a data breach. To secure your accounts, Stead Vault recommends checking your saved passwords.
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHANGE_PASSWORD_MESSAGE_GPM_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials are not saved for other sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, branded.">
-    The password you just used was found in a data breach. Google Password Manager recommends changing your password now.
+    The password you just used was found in a data breach. Stead Vault recommends changing your password now.
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHANGE_AND_CHECK_PASSWORDS_MESSAGE_GPM_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials were not saved but used on multiple sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, branded.">
-    The password you just used was found in a data breach. To secure your accounts, Google Password Manager recommends changing it now and checking your saved passwords.
+    The password you just used was found in a data breach. To secure your accounts, Stead Vault recommends changing it now and checking your saved passwords.
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHECK_PASSWORDS_MESSAGE_GPM_NON_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials are saved and used on multiple sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, non branded.">
-    The password you just used was found in a data breach. To secure your accounts, Password Manager recommends checking your saved passwords.
+    The password you just used was found in a data breach. To secure your accounts, Stead Vault recommends checking your saved passwords.
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHANGE_PASSWORD_MESSAGE_GPM_NON_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials are not saved for other sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, non branded.">
-    The password you just used was found in a data breach. Password Manager recommends changing this password now.
+    The password you just used was found in a data breach. Stead Vault recommends changing this password now.
   </message>
   <message name="IDS_CREDENTIAL_LEAK_CHANGE_AND_CHECK_PASSWORDS_MESSAGE_GPM_NON_BRANDED" desc="The text that is used in credential leak detection dialog when the leaked credentials were not saved but used on multiple sites. The leaked credentials may have been leaked by the current website, some other third-party website or even a third-party app used by the user. It could also be coincidental reuse of a trivial password used by some other users in the world and exposed in a public leak. Google Password Manager updated phrasing, non branded.">
-    The password you just used was found in a data breach. To secure your accounts, Password Manager recommends changing it now and then checking your saved passwords.
+    The password you just used was found in a data breach. To secure your accounts, Stead Vault recommends changing it now and then checking your saved passwords.
   </message>
   <if expr="is_ios">
     <message name="IDS_IOS_PASSKEY_SUGGESTION_LABEL" desc="Subtitle of a passkey suggestion to differentiate password and passkey suggestions in the different autofill surfaces. [iOS only]">
@@ -84,7 +84,7 @@
     Saved passwords
   </message>
   <message name="IDS_PASSWORD_MANAGER_DEFAULT_EXPORT_FILENAME" desc="Chrome suggests this file name when user chooses to export their passwords saved with Chrome." formatter_data="android_java">
-    Chrome Passwords
+    Stead Vault Passwords
   </message>
   <message name="IDS_PASSWORD_MANAGER_PASSWORD_FOR_ACCOUNT" desc="Voice over text read if the user focuses a drop down entry to fill a password for a given account.">
     Password for <ph name="username">$1<ex>chef@google.com</ex></ph>
@@ -105,7 +105,7 @@
     Passkey • iCloud Keychain
   </message>
   <message name="IDS_PASSWORD_MANAGER_PASSKEY_FROM_GOOGLE_PASSWORD_MANAGER" desc="Text that appears in the password autofill popup when listing a passkey from Google Password Manager. When the user clicks on this option, Chrome will sign the user into the site with this passkey, which may involve prompts for the user to prove their identity via biometrics or by entering a PIN. Refer to Glossary for the meaning of 'passkey'; do not translate it as 'password'.">
-    Passkey • Google Password Manager
+    Passkey • Stead Vault
   </message>
   <message name="IDS_PASSWORD_MANAGER_PASSKEY_FROM_WINDOWS_HELLO" desc="Text that appears in the password autofill popup, for a Windows Hello passkey. When the user clicks on this option, Chrome will invoke Windows Hello to verify the user's identity. Windows Hello is a Microsoft brand and should be translated the same as it is in Windows. If you edit the language code in https://support.microsoft.com/en-us/windows/learn-about-windows-hello-and-set-it-up-dae28983-8242-bb2a-d3d1-87c9d265a5f0 you can see how Microsoft translates it in several languages (It is often left untranslated). Refer to Glossary for the meaning of 'passkey'; do not translate it as 'password'.">
     Passkey • Windows Hello
@@ -114,7 +114,7 @@
     Passkey • <ph name="PROVIDER_NAME">$1<ex>Apple Passwords</ex></ph>
   </message>
   <message name="IDS_PASSWORD_MANAGER_PASSWORD_FROM_GOOGLE_PASSWORD_MANAGER" desc="Text that appears for a password on a dialog that can display both passwords and passkeys for the user to select.">
-    Password • Google Password Manager
+    Password • Stead Vault
   </message>
 
   <if expr="use_titlecase">
@@ -219,30 +219,30 @@
       You can change whether you use your screen lock for filling passwords in <ph name="SETTINGS">$1<ex>Settings</ex></ph>
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_TITLE_BRANDED" desc="Text that appears in the title of the relaunch Chrome bubble.">
-      Google Password Manager needs access to MacOS Keychain
+      Stead Vault needs access to macOS Keychain
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_BRANDED" desc="Text that appears in the description of the relaunch Chrome bubble.">
-      To use Google Password Manager with macOS Keychain, relaunch Chrome and allow Keychain access. Your tabs will reopen after relaunching.
+      To use Stead Vault with macOS Keychain, relaunch Stead and allow Keychain access. Your tabs will reopen after relaunching.
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_TITLE_NON_BRANDED" desc="Text that appears in the title of the relaunch Chromium bubble.">
-      Password Manager needs access to MacOS Keychain
+      Stead Vault needs access to macOS Keychain
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with macOS Keychain, relaunch Chromium and allow Keychain access. Your tabs will reopen after relaunching.
+      To use Stead Vault with macOS Keychain, relaunch Stead and allow Keychain access. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_linux">
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_TITLE_LINUX_BRANDED" desc="Text that appears in the title of the relaunch Chrome bubble.">
-      Google Password Manager needs more access
+      Stead Vault needs more access
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_LINUX_BRANDED" desc="Text that appears in the description of the relaunch Chrome bubble.">
-      To use Google Password Manager with your operating system, relaunch Chrome and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+      To use Stead Vault with your operating system, relaunch Stead and allow access to your computer's password manager. Your tabs will reopen after relaunching.
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_TITLE_LINUX_NON_BRANDED" desc="Text that appears in the title of the relaunch Chromium bubble.">
-      Password Manager needs more access
+      Stead Vault needs more access
     </message>
     <message name="IDS_PASSWORD_MANAGER_RELAUNCH_CHROME_BUBBLE_DESCRIPTION_LINUX_NON_BRANDED" desc="Text that appears in the description of the relaunch Chromium bubble.">
-      To use Password Manager with your operating system, relaunch Chromium and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+      To use Stead Vault with your operating system, relaunch Stead and allow access to your computer's password manager. Your tabs will reopen after relaunching.
     </message>
   </if>
   <if expr="is_win or is_macosx or is_chromeos">
@@ -316,10 +316,10 @@
     Try this recovery password
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_PROACTIVE_RECOVERY_FOOTER_BRANDED" desc="Footer text of an autofill suggestion drop-down that explains why we are suggesting to try a recovery password.">
-    You recently changed a password found in a public data breach. In case of trouble, Google Password Manager can help you sign in.
+    You recently changed a password found in a public data breach. In case of trouble, Stead Vault can help you sign in.
   </message>
   <message name="IDS_PASSWORD_MANAGER_UI_PROACTIVE_RECOVERY_FOOTER_NON_BRANDED" desc="Footer text of an autofill suggestion drop-down that explains why we are suggesting to try a recovery password.">
-    You recently changed a password found in a public data breach. In case of trouble, Password Manager can help you sign in.
+    You recently changed a password found in a public data breach. In case of trouble, Stead Vault can help you sign in.
   </message>
   <if expr="is_android">
     <then>
