--- a/chrome/app/generated_resources.grd
+++ b/chrome/app/generated_resources.grd
@@ -11539,7 +11539,7 @@
           To save your password to Stead Vault, click the key
         </message>
         <message name="IDS_PASSWORDS_SAVE_RECOVERY_PROMO_SCREENREADER" is_accessibility_with_no_ui="true" desc="Screenreader text for IPH prompting users to reconsider saving passwords">
-          To save your password to Stead Vault, the Password Manager icon in the Location Bar
+          To save your password to Stead Vault, the Stead Vault icon in the Location Bar
         </message>
         <message name="IDS_PASSWORD_MANAGER_TOOLTIP_BLOCKED" desc="Text for the 'Manage Passwords' when the user has blocked saving passwords for this site">
           Never for this site
--- a/chrome/app/password_manager_ui_strings.grdp
+++ b/chrome/app/password_manager_ui_strings.grdp
@@ -533,7 +533,7 @@
   </then>
   <else>
     <message name="IDS_PASSWORD_MANAGER_UI_CHANGE_PASSWORD_MANAGER_PIN" desc="The title of a toggle which allows users to change the password manager PIN.">
-      Change Password Manager PIN
+      Change Stead Vault PIN
     </message>
   </else>
   </if>
@@ -887,10 +887,10 @@
     </then>
     <else>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_TITLE" desc="Title for not branded Relaunch Chromium promo card.">
-        Password Manager needs access to MacOS Keychain
+        Stead Vault needs access to macOS Keychain
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION" desc="Description for not branded Relaunch ChrChromiumome promo card.">
-        To use Password Manager with macOS Keychain, relaunch Chromium and allow Keychain access. Your tabs will reopen after relaunching.
+        To use Stead Vault with macOS Keychain, relaunch Stead and allow Keychain access. Your tabs will reopen after relaunching.
       </message>
     </else>
     </if>
@@ -907,10 +907,10 @@
     </then>
     <else>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_TITLE_LINUX" desc="Title for the Relaunch Chromium promo card.">
-        Password Manager needs more access
+        Stead Vault needs more access
       </message>
       <message name="IDS_PASSWORD_MANAGER_UI_RELAUNCH_CHROME_PROMO_CARD_DESCRIPTION_LINUX" desc="Description for the Relaunch Chromium promo card.">
-        To use Password Manager with your operating system, relaunch Chromium and allow access to your computer's password manager. Your tabs will reopen after relaunching.
+        To use Stead Vault with your operating system, relaunch Stead and allow access to your computer's password manager. Your tabs will reopen after relaunching.
       </message>
     </else>
     </if>
